modules/openapi-generator/src/main/resources/Java/libraries/native/pojo.mustache

```diff
@@ -1,6 +1,7 @@
 {{#discriminator}}
 import {{invokerPackage}}.JSON;
 {{/discriminator}}
+import {{invokerPackage}}.ApiClient;
```

Comment:
can we wrap it as follows?
```
{{#supportUrlQuery}}
import {{invokerPackage}}.ApiClient;
{{/supportUrlQuery}}
```

Reply:
I don't see why not. I didn't see any changes to the samples from this change.

Reply:
I don't see why not. I didn't see any changes to the samples from this change.

Reply:
it helps to avoid blank line break